portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -38,6 +40,22 @@ public DataServlet() {
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
         // Delete everything from previous step.
+        Query query = new Query("Task");
+        PreparedQuery results = datastore.prepare(query);
+
+        ArrayList<String> commentList = new ArrayList<String>();
+        for (Entity entity : results.asIterable()) {
```

Comment:
optional: you could condense the contents of this loop a bit by just saying
commentList.add((String) entity.getProperty("comment-input"));